### PR TITLE
Ratings Component

### PR DESCRIPTION
**Pull request by sahiba0915**
Files to check:

1. ratings.html
2. ratings.css

live preview: https://ratingscomp.netlify.app/component/ratings/ratings.html

### CODE REVIEW

File: component/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  --btn-color: #965c49;
+  --box-shadow: #979494;
+  --primary-font: "Poppins", sans-serif;
+  --primary-alert: #80b918;
+  --success-alert: #dee2e6;
+  --info-alert: #4cc9f0;
+  --warning-alert: #d62828;
+  --default-color: #000000;
+  --pink: #e76f51;
+  --border: rgba(0, 0, 0, 0.3);
```
Comment (akash-1318) [inline component/navigation/navigation.css:21]: Don't change color format here also use hex color code.

File: component/ratings/ratings.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  --btn-color: #965c49;
+  --box-shadow: #979494;
+  --primary-font: "Poppins", sans-serif;
+  --primary-alert: #80b918;
+  --success-alert: #dee2e6;
+  --info-alert: #4cc9f0;
+  --warning-alert: #d62828;
+  --default-color: #000000;
+  --pink: #e76f51;
+  --border: rgba(0, 0, 0, 0.3);
```
Comment (akash-1318) [inline component/ratings/ratings.css:21]: Use hex color code here also.

File: component/slider/slider.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  --btn-color: #965c49;
+  --box-shadow: #979494;
+  --primary-font: "Poppins", sans-serif;
+  --primary-alert: #80b918;
+  --success-alert: #dee2e6;
+  --info-alert: #4cc9f0;
+  --warning-alert: #d62828;
+  --default-color: #000000;
+  --pink: #e76f51;
+  --border: rgba(0, 0, 0, 0.3);
```
Comment (akash-1318) [inline component/slider/slider.css:21]: Same change here as well.

**Review by akash-1318 — COMMENTED** [review]
Just look into the suggestions i added besides that your html code is really clean with proper comments so you can continue to merge.

File: component/navigation/navigation.css (hunk 1 above)
Comment (sahiba0915) [reply component/navigation/navigation.css:21]: will work on that thanks

File: component/ratings/ratings.css (hunk 2 above)
Comment (sahiba0915) [reply component/ratings/ratings.css:21]: ok

File: component/slider/slider.css (hunk 3 above)
Comment (sahiba0915) [reply component/slider/slider.css:21]: ok

**Review by chanchal-panpaliya — COMMENTED** [review]
code looks good to me .

File: component/badge/badge.html [hunk 4]
```diff
@@ -39,12 +39,13 @@
         <p class="sidebar-heading">Components</p>
         <ul class="sidebar-list">
           <li><a href="/component/avatar/avatar.html">Avatars</a></li>
-          <li><a href="/component/badge/badge.html">Badges</a></li>
+          <li><a href="/component/badge/badge.html" class="active">Badges</a></li>
```
Comment (pariyar07) [inline component/badge/badge.html:42]: It's good you have made a single class and used where applicable

Comment (sahiba0915) [reply component/badge/badge.html:42]: thanks

File: component/lists/list.css [hunk 5]
```diff
@@ -0,0 +1,58 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: var(--primary-font);
```
Comment (pariyar07) [inline component/lists/list.css:5]: It's not against the rules in using * selector which means any/all elements, but will obviously be on the bottom of the property chain when it comes to overriding more specific selectors. I guess the best practice is to use inside body.

**Comment by pariyar07** [comment]
Code looks good to me, just make a few changes that I have suggested. And next time try to add a screenshot or a short video for the better understanding of the UI.

**Pull request merged**